modules/cluster-logging-rn-5.5.3.adoc

```diff
@@ -0,0 +1,36 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-3_{context}"]
+= Logging 5.5.3
+This release includes link:https://access.redhat.com/errata/RHBA-2022:6858[OpenShift Logging Bug Fix Release 5.5.3].
+
+[id="openshift-logging-5-5-3-bug-fixes_{context}"]
+== Bug fixes
+* Before this update log entries that had structured messages included the original message field which made the entry larger. This update removes the message field for structured logs to reduce the increased size. (link:https://issues.redhat.com/browse/LOG-2759[LOG-2759])
```

Comment:
Missing comma:

`Before this update, log entries that had structured messages...`